### ISSUE TITLE
 Code blocks and syntax highlighting no longer works

### ISSUE THREAD

**Issue by pakyrs** [dracula/obsidian#45]
I believe this happened before but all the code blocks are now no longer showing and the syntax highlighting is also not functioning properly.

![2024-06-12_08-41](https://github.com/dracula/obsidian/assets/56080974/53af71fb-bda6-473d-9067-2ba1c4a9e6d7)

**Comment by Chrismettal** [comment]
Can you share your exact string?

Code block highlighting is working generally, but it might be missing highlighting for shell scripts (You can see "vim" below being highlighted at least)

Other themes I'm flicking through also don't do much with the install commands specifically.

![image](https://github.com/dracula/obsidian/assets/39636046/b2a7f954-884c-4b00-a648-eeeb8d8902b9)

**Comment by pakyrs** [comment]
Indeed the simple shell commands I have don't pay any justice, however not even the comments are commented out.

This was working absolutely as expected until the theme updated.

As you can see the highlighting works, but commented lines are not grey or less obvious. Also weirdly the code block is the same color as the background of the page, this only happen in this theme, and I don't have any weird CSS going on.

Dracula Official
![2024-06-13_21-18](https://github.com/dracula/obsidian/assets/56080974/b6bf1185-fa39-48c2-81da-9ea9371e3132)

Dracula Gemini
![2024-06-13_21-21](https://github.com/dracula/obsidian/assets/56080974/bf282e50-5ffc-4f1c-bbbb-c17749fb131f)

Any ideas? 

To further troubleshoot I spin up a Fedora VM and installed the latest Obsidian Flatpak and I am surprised to see that Dracula Theme doesn't show in the store?

![2024-06-13_21-23](https://github.com/dracula/obsidian/assets/56080974/811f0530-1b93-4c5e-8a22-0ced07a9212e)

**Comment by pakyrs** [comment]
Ok i found the pickle, I don't know CSS, but the version pushed on Obsidian theme store is 1.1.0 which should be the same as the git repo, however the code for theme.css differ.

Therefore 1.0.7 was my original which had no issue, I was able to grab this from my ZFS snapshots and everything started to work again.

Then comparing 1.1.0 from git vs 1.1.0 from the obsidian store they are different. I downloaded the git version and it works fine, hence the issue is in the code pushed to Obsidian then.

**Comment by Chrismettal** [comment]
So 1.1.0 straight from the repo works for you?

There's gonna be a small rework to code theming soon (https://github.com/dracula/obsidian/issues/39) so a new version will be pushed to the store afterwards.

**Comment by pakyrs** [comment]
Yes the one from repo works fine, it's the one in the store that differ slightly and no longer works on code bloc highligting. Thanks for letting me know, for the moment I just manually replaced the css file.

**Comment by TheFrenchGhosty** [comment]
It's been 3 months and the problem is still present in the store.

I had to find this issue to figure out that the weird way the code blocks were rendering were a bug...

**Comment by Chrismettal** [comment]
Fetching the current version through Obsidian renders code blocks correctly

![Image](https://github.com/user-attachments/assets/89412754-55cb-41c3-ac80-f8f7a9aa90bb)
